src/components/MenuButton/MenuButton.jsx

```diff
@@ -139,6 +141,8 @@ const MenuButton = ({
   const onMouseUp = event => {
     if (disabled) {
       event.currentTarget.blur();
+    } else {
+      isFunction(onClick) && onClick();
```

Comment:
```
if(disabled) {
      event.currentTarget.blur();
     return
}

onClick(event);
```

Reply:
I'll add the default NOOP instead of the isFunction() check?